PatternFinding.py

```diff
@@ -129,11 +126,9 @@ def CheckingRatioOfContours(self, index):
         # print areaofsecondchild
         # print (areaoffirst/areaofsecondchild)
 
-        if (areaoffirst / areaofsecondchild) > 1 and \
-           ((areaoffirst / areaofsecondchild) < 10):
-            return True
-        else:
-            return False
+        return not ((areaoffirst / areaofsecondchild) > 1 and \
```

Comment:
` return not ` is used if the statement is true but output needs to be false, which is not the case here.
Only ` return ` would do here.